--- a/build/config/win/compiler.gni
+++ b/build/config/win/compiler.gni
@@ -123,7 +123,7 @@
     # microarchitecture. MSVC only supports a subset of architectures, and the
     # next step after SSE2 will be AVX.
     if (current_cpu == "x86" || current_cpu == "x64") {
-      cflags += [ "-msse3" ]
+      cflags += [ "-mavx512" ]
     }
 
     # Enable ANSI escape codes if something emulating them is around (cmd.exe
