--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -800,7 +800,7 @@
 # [0]: https://pinpoint-dot-chromeperf.appspot.com/job/16984a18be0000
 # [1]: https://pinpoint-dot-chromeperf.appspot.com/job/11984a18be0000
 # [2]: https://ci.chromium.org/ui/p/chromium/builders/try/android-binary-size/1848442
-import_instr_limit = 30
+import_instr_limit = 100
 
 if (is_win) {
   ldflags += [
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1188,7 +1188,7 @@
   if (!is_debug && use_thin_lto && is_a_target_toolchain) {
     if (thin_lto_enable_optimizations) {
-      lto_opt_level = 2
+      lto_opt_level = 3
     } else {
       lto_opt_level = 0
     }
