--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -803,8 +803,8 @@
-  import_instr_limit = 30
+  import_instr_limit = 100
   if (is_win) {
     ldflags += [
       "/opt:lldltojobs=all",
       "-mllvm:-import-instr-limit=$import_instr_limit",
       "-mllvm:-disable-auto-upgrade-debug-info",
     ]
   } else {
@@ -1191,7 +1191,7 @@
   if (!is_debug && use_thin_lto && is_a_target_toolchain) {
     if (thin_lto_enable_optimizations) {
-      lto_opt_level = 2
+      lto_opt_level = 3
     } else {
       lto_opt_level = 0
     }
